tutorials/mandels_problem.ipynb

```diff
@@ -168,76 +168,20 @@
    },
    "outputs": [
     {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "\n"
+     "ename": "NameError",
+     "evalue": "name 'solver_params' is not defined",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
```

Comment:
Could you revert the changes here (Error does not look nice :) ).

Reply:
The tutorial runs without error, can you please check again.